### ISSUE TITLE
Documentation focus ring is nearly invisible in Firefox

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri-design-system#206]
## Product

Design system docs

## Expected behavior

Focus ring should be easily visible in all browsers

## Actual behavior

Visible in Chrome but not Firefox on macOS:

| Chrome | Firefox |
|--|--|
| ![2021-03-25 09 10 09](https://user-images.githubusercontent.com/2367265/112505832-649c6700-8d4a-11eb-96e2-3c9ea9218c07.gif) | ![2021-03-25 09 10 25](https://user-images.githubusercontent.com/2367265/112505845-66fec100-8d4a-11eb-8625-696cb6f62ccb.gif) |

**Comment by nikkuAg** [comment]
Is this issue still open? I would like to work on this

**Comment by MisRob** [comment]
Hi @nikkuAg, since we haven't seen activity here for a long time, I will assign you, If you're still interested, you're welcome to messages us and we can assign again. Be well

**Comment by RONAK-AI647** [comment]
Hi @indirectlylit @AlexVelezLl I want an opportunity to work on it , will you please assign it to me ,thank you.

**Comment by AlexVelezLl** [comment]
Hey @RONAK-AI647! For sure! I am assigning this to you. Let us know if you have any question :)

**Comment by RONAK-AI647** [comment]
Hey team members @AlexVelezLl @rtibbles @indirectlylit ,there are some queries in my head ,would be pleased if I get suggestions.

**1)I don't see any focus ring in my system when hover around the documentation.**

https://github.com/user-attachments/assets/63ced118-9a09-437e-bc90-bf037ccb88ba


**2) If the first one is my system error ,then increasing the width of the focus ring is sufficient???
3) Are you sure this is not individual's browser issue? In my case both the browsers gives the same result.**

**Comment by AllanOXDi** [comment]
Hi @RONAK-AI647 

       > 1)I don't see any focus ring in my system when hover around the documentation.
Could you try navigating using a tab key?

**Comment by RONAK-AI647** [comment]
@indirectlylit @MisRob kindly take a look at my PR .

**Comment by MisRob** [comment]
Thanks @RONAK-AI647!

**Comment by MisRob** [comment]
Closed by https://github.com/learningequality/kolibri-design-system/pull/827
